x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -60,8 +61,8 @@ import { FieldInput } from './field_input';
 import { NameInput } from '../../shared_components';
 import { ParamEditorProps } from '../operations/definitions';
 import { WrappingHelpPopover } from '../help_popover';
-
-const operationPanels = getOperationDisplay();
```

Comment:
Why did you pull this deeper into the functions? It seems like it's completely static anyway

Reply:
I couldn't make the tests work with the current setup :( I memoized the function so I think that solves the problem with unnecessary re-calculation. Let me know if you have a better idea.

Reply:
What test is problematic?

Reply:
It's actually only a problem for `ReferenceEditor` - the last commit I just pushed shows it - around 40 tests suites break with the same:
<img width="579" alt="Screenshot 2022-06-29 at 11 12 13" src="https://user-images.githubusercontent.com/4283304/176399383-018f1b01-ed36-4e96-8ab1-f49a0b9f014c.png">

It has something to do with importing reference editor inside the terms operation definition, but I cannot figure it out :(

Reply:
Actually, it's not just tests, Lens also doesn't load properly with: 
`TypeError: Object(...) is not a function  at ./public/indexpattern_datasource/dimension_panel/reference_editor.tsx`